tests/test_sca.py

```diff
@@ -167,66 +171,81 @@ def test_download_lookaside_sources(
     (
         1890406,
-        "grafana-container",
+        True,
+        "",
+        "",
         "",
         "tests/data/grafana-syft.json",
         "pkg:npm/acorn-globals@4.3.4",
     ),
     (
         1888203,
-        "cnf-tests-container",
+        True,
+        "",
+        "",
         "",
         "tests/data/cnf-tests-syft.json",
         "pkg:pypi/requests@2.26.0",
     ),
 ]
 
 
-def mock_clone(package_name: str, package_type: str) -> Tuple[Optional[Path], str]:
-    package_name = package_name.removesuffix("-container")
-    return Path("tests/data") / package_type / package_name, package_name
+def mock_clone(package_name: str, build_id: int) -> Tuple[Path, str, str]:
+    target_path = Path(f"/tmp/{build_id}")
+    if not target_path.exists():
+        os.mkdir(f"/tmp/{build_id}")
+    # Copy test files into scratch location
+    test_sources = Path(f"tests/data/{build_id}/sources")
+    if test_sources.exists():
+        shutil.copyfile(test_sources, target_path / "sources")
+    # TODO split package_type out to testdata
+    package_type = "rpms"
+    return target_path, package_type, package_name
 
 
 @pytest.mark.parametrize(
-    "build_id,package_name,lookaside_tarfile,syft_results,expected_purl",
+    "build_id,is_container,package_name,lookaside_file,download_path,syft_results,expected_purl",
     slow_software_composition_analysis_test_data,
 )
 # mock the syft call to avoid having to have actual source code for the test
 @patch("subprocess.check_output")
 @patch("corgi.tasks.sca._clone_source", side_effect=mock_clone)
+@patch("corgi.core.models.SoftwareBuild.save_product_taxonomy")
 def test_slow_software_composition_analysis(
+    mock_save_prod_tax,
     mock_clone_source,
     mock_syft,
     build_id,
+    is_container,
     package_name,
-    lookaside_tarfile,
+    lookaside_file,
+    download_path,
     syft_results,
     expected_purl,
+    requests_mock,
 ):
-    sb = SoftwareBuildFactory(
-        build_id=build_id,
-        name=package_name,
-        source=package_name,
-    )
-    is_container = package_name.endswith("-container")
+    sb = SoftwareBuildFactory(build_id=build_id, source=package_name)
     if is_container:
         root_component = ComponentFactory(
-            type=Component.Type.CONTAINER_IMAGE, software_build=sb, name=package_name, arch="noarch"
+            type=Component.Type.CONTAINER_IMAGE, software_build=sb, arch="noarch"
         )
     else:
         root_component = ComponentFactory(
-            type=Component.Type.SRPM, software_build=sb, name=package_name
+            type=Component.Type.SRPM,
+            software_build=sb,
         )
     root_component.cnodes.get_or_create(
         type=ComponentNode.ComponentNodeType.SOURCE, parent=None, purl=root_component.purl
     )
     assert not Component.objects.filter(purl=expected_purl).exists()
+
+    expected_url = f"https://{os.getenv('CORGI_LOOKASIDE_CACHE_URL')}/repo/{download_path}"
```

Comment:
Maybe I'm confused, but didn't we get rid of this setting? Or if I'm thinking of a different setting, let's keep all the related options in one place (either as constants in sca.py, or as settings in base.py).